hack/triage/config.yaml

```diff
@@ -0,0 +1,278 @@
+      - triage-lifecycle-frozen
+      - triage-all
+
+  - id: actionable
+    name: Actionable
+    description: >
+      Issues that can be worked on.
+    dedup: false
+    rules:
+      - actionable-not-assigned-without-help-or-good-first-issue
+      - actionable-updated
+      - actionable-all
+
+  - id: fix-me
+    name: Fix-me!
+    description: >
+      Ensure we are using the labels in a consistent way.
+    dedup: false
+    rules:
+      - fixme-avoid-triage-needs-information
+      - fixme-avoid-kind-design
+      - fixme-kind-proposal-without-kind-feature
+      - fixme-kind-support-with-other-kinds
+      - fixme-triage-accepted-with-kind-support-or-priority-awaiting-more-evidence
+      - fixme-help-or-good-first-issue-before-triage-accepted
+      - fixme-lifecycle-frozen-with-priority-critical-urgent-priority-important-soon-priority-important-longterm
+
+rules:
+  # Phase: Initial triage
+
+  triage-needs-kind:
+    # WHY?: kinds need to be assigned by maintainers, let's do it!
+    name: "needs-kind"
+    resolution: "Add a kind/ label: api-change, bug, cleanup, deprecation, documentation, failing-test, feature, flake, proposal, regression, release-blocking, support"
+    type: issue
+    filters:
+      # TODO: Configure plugins so needs-kind label is applied + use it for this filter
+      - label: "!kind/.*"
+
+  triage-needs-priority:
+    # WHY?: kinds need to be assigned by maintainers, let's do it!
+    name: "needs-priority"
+    resolution: "Add a priority/ label: priority/critical-urgent > priority/important-soon > priority/important-longterm > priority/backlog > priority/awaiting-more-evidence"
+    type: issue
+    filters:
+      # TODO: Configure plugins so needs-priority label is applied + use it for this filter
+      - label: "!priority/.*"
+
+  # Phase: Triage finalization
+
+  triage-stale-awaiting-more-evidence:
+    # WHY?: issue authors are supposed to provide answers timely when an issue is labeled priority/awaiting-more-evidence, otherwise /close
+    name: "priority/awaiting-more-evidence not commented in the last 15 days"
+    resolution: "Consider if to close with /close or to remove priority/awaiting-more-evidence label"
+    type: issue
+    filters:
+      - label: "!triage/accepted"
+      - label: "priority/awaiting-more-evidence"
+      - commented: +15d
+
+  triage-stale-kind-support:
+    # WHY?: support request are supposed to be quick back and forth between authors and maintainers, otherwise /close
+    name: "Support request not commented in the last 15 days"
+    resolution: "Consider if to close with /close or if to change kind"
+    type: issue
+    filters:
+      - label: "!triage/accepted"
+      - label: "kind/support"
+      - commented: +15d
+
+  triage-updated:
+    # WHY?: maintainers are supposed to keep up with issue discussions and help to move to actionable
+    # Note: maintainers can't make everything actionable (so it is okay if this list does not go to 0).
```

Comment:
Optional nit: Wondering if we should add this to the description of the "triage-refinement"  collection son its visible when running triage-party.